pkg/schedule/schedule.go

```diff
@@ -290,6 +290,77 @@ func Init() error {
 		}
 	}
 
+	return createDefaultPolicy()
+}
+
+func createDefaultPolicy() error {
+	_, err := k8s.Instance().CreateSchedulePolicy(&stork_api.SchedulePolicy{
+		ObjectMeta: meta.ObjectMeta{
+			Name: "default-migration-policy",
```

Comment:
should we call it migration policy or keep it generic?

Reply:
This is used specifically as the default for migrations when creating a schedule using `storkctl`